cmd/operator/deploy/operator/03-role.yaml

```diff
@@ -41,38 +41,60 @@ rules:
 - apiGroups: [""]
   resources:
   - secrets
-  verbs: ["create", "patch", "update"]
+  verbs: ["list", "watch", "create"]
+- apiGroups: [""]
+  resources:
+  - secrets
+  resourceNames: ["collection", "rules"]
+  verbs: ["get", "patch", "update"]
 - apiGroups: [""]
   resources:
   - configmaps
-  verbs: ["create", "patch", "update"]
+  verbs: ["list", "watch", "create"]
+- apiGroups: [""]
+  resources:
+  - configmaps
+  resourceNames: ["collector", "rule-evaluator", "rules-generated"]
+  verbs: ["get", "patch", "update"]
 - apiGroups: ["apps"]
   resources:
   - daemonsets
-  verbs: ["create", "delete", "patch", "update"]
+  resourceNames: ["collector"]
+  verbs: ["get", "list", "watch", "delete", "patch", "update"]
```

Comment:
I believe you will need to break this rule similar to what you did for the others?
```
- apiGroups: ["apps"]
   resources:
   - daemonsets
   resourceNames: ["collector"]
   verbs: ["get", "delete", "patch", "update"]
- apiGroups: ["apps"]
   resources:
   - daemonsets
   verbs: ["list", "watch"]
  ```

Reply:
You can watch on a single resource name or all but not any other discrete value. I had to break up the other rules because they had more than 1 `resourceNames`. That's why in this same commit I made a code change to watch only the `collector` `daemonset` resource name.

Reply:
oh gotcha! thanks!